lib/slack/bot.ex

```diff
@@ -87,11 +105,11 @@ defmodule Slack.Bot do
       token: token,
       me: rtm.self,
       team: rtm.team,
-      bots: rtm_list_to_map(rtm.bots),
-      channels: rtm_list_to_map(rtm.channels),
-      groups: rtm_list_to_map(rtm.groups),
-      users: rtm_list_to_map(rtm.users),
-      ims: rtm_list_to_map(rtm.ims)
+      bots: bot_to_map(Bots.info(%{token: token}) |> Map.get("bot")),
```

Comment:
Currently `bots.info` returns an empty `{ok: true}` response if `bot` argument is absent. As a result, `bot_to_map` fails.

Reply:
Is this a permissions problem because `Bots.info` requires you have the `bot` permission enabled for your app?

Reply:
I was testing with a bot token on an app that doesn't have access to this scope via generic one, just in case. It could be a temporary API bug, but I don't have good way to check for that.